docs/reference/ccr/apis/follow/post-forget-follower.asciidoc

```diff
@@ -0,0 +1,152 @@
+performing some operations on the leader index. This API exists to enable
+manually removing these retention leases when the unfollow API was unable to do
+so.
+
+NOTE: This API does not stop replication by a following index. If you use this
+API targeting a follower index that is still actively following, the following
+index will add back retention leases on the leader. The only purpose of this API
+is to handle the case of failure to remove the following retention leases after
+the <<ccr-post-unfollow,unfollow API>> is invoked.
+
+==== Request
+
+//////////////////////////
+
+[source,js]
+--------------------------------------------------
+PUT /follower_index/_ccr/follow?wait_for_active_shards=1
+{
+  "remote_cluster" : "remote_cluster",
+  "leader_index" : "leader_index"
+}
+--------------------------------------------------
+// CONSOLE
+// TESTSETUP
+// TEST[setup:remote_cluster_and_leader_index]
+
+[source,js]
+--------------------------------------------------
+POST /follower_index/_ccr/pause_follow
+--------------------------------------------------
+// CONSOLE
+// TEARDOWN
+
+//////////////////////////
+
+[source,js]
+--------------------------------------------------
+POST /<leader_index>/_ccr/forget_follower
+{
+  "follower_cluster" : "<follower_cluster>",
+  "follower_index" : "<follower_index>",
+  "follower_index_uuid" : "<follower_index_uuid>",
+  "leader_remote_cluster" : "<leader_remote_cluster>"
+}
+--------------------------------------------------
+// CONSOLE
+// TEST[s/<leader_index>/leader_index/]
+// TEST[s/<follower_cluster>/follower_cluster/]
+// TEST[s/<follower_index>/follower_index/]
+// TEST[s/<follower_index_uuid>/follower_index_uuid/]
+// TEST[s/<leader_remote_cluster>/leader_remote_cluster/]
+// TEST[skip_shard_failures]
+
+[source,js]
+--------------------------------------------------
+{
+  "_shards" : {
+    "total" : 1,
+    "successful" : 1,
+    "failed" : 0,
+    "failures" : [ ]
+  }
+}
+--------------------------------------------------
+// TESTRESPONSE[s/"total" : 1/"total" : $body._shards.total/]
+// TESTRESPONSE[s/"successful" : 1/"successful" : $body._shards.successful/]
+// TESTRESPONSE[s/"failed" : 0/"failed" : $body._shards.failed/]
+// TESTRESPONSE[s/"failures" : \[ \]/"failures" : $body._shards.failures/]
+
+==== Path Parameters
+
+`leader_index` (required)::
+  (string) the name of the leader index
+
+==== Request Body
+`follower_cluster` (required)::
+  (string) the name of the cluster containing the follower index
+
+`follower_index` (required)::
+  (string) the name of the follower index
+
+`follower_index_uuid` (required)::
+  (string) the UUID of the follower index
+
+`leader_remote_cluster` (required)::
+  (string) the alias (from the perspective of the cluster containing the
+  follower index) of the <<modules-remote-clusters,remote cluster>> containing
+  the leader index
+
+==== Authorization
+
+If the {es} {security-features} are enabled, you must have `manage_follow_index`
```

Comment:
The `IndexPrivilege#MANAGE_FOLLOW_INDEX_AUTOMATON` constant isn't updated to include the forget follower api. Also this text should say: `index privileges for the leader index.` instead of `index privileges for the follower index.`.

I also I wonder whether manage_follow_index is the right privilege here. Because this is about a leases for the follower index in the leader index. I don't know a good alternative here. I just wanted to mention this.

Reply:
I pushed e68b0c091c1785fe8de0688757c01ed463b1837c.